app/HutchManager/Dockerfile

```
@@ -1,9 +1,11 @@
 ﻿FROM node:lts-alpine as spa_build
+COPY pnpm-lock.yaml .
+COPY pnpm-workspace.yaml .
+COPY package.json .
+COPY ./app/manager-frontend /manager-frontend
 RUN npm install -g pnpm
-WORKDIR /manager-frontend
-COPY ../manager-frontend .
 RUN pnpm install --frozen-lockfile
```

Should filter to the package we're interested in only

```suggestion
RUN pnpm install --filter manager-frontend --frozen-lockfile
```